test/lib/jdk/test/lib/artifacts/ArtifactResolverException.java

```diff
@@ -1,3 +1,26 @@
+/*
+ * Copyright (c) 2018, Oracle and/or its affiliates. All rights reserved.
```

Comment:
The file was edited in 2020.
```suggestion
 * Copyright (c) 2018, 2020, Oracle and/or its affiliates. All rights reserved.
```

Reply:
Good catch!  Added the updated year.  Thank you!